packages/components/src/Gallery/GalleryTypes.d.ts

```diff
@@ -0,0 +1,70 @@
+import { IconNames } from "../Icon";
+
+export type Sizes = "small" | "base" | "large" | "extraLarge";
+
+export type FileIconsNames = Extract<
+  IconNames,
+  "file" | "excel" | "pdf" | "video" | "word"
+>;
+
+export interface GalleryProps {
+  /**
+   * The size of the Gallery and it's files
+   * @default "base"
+   */
+  size?: "base" | "large";
+
+  /**
+   * The files for the Gallery to display
+   */
+  files: File[];
+
+  /**
+   * The max number of thumbnails before no more thumbnails are displayed
+   * unless the user clicks an action to display the rest
+   */
+  max?: number;
+
+  /**
+   * onDelete callback - this function will be called when the delete action is triggered on a Gallery image
```

Comment:
nit: ah! you were doing so good on the previous doc string!! 😆 

```suggestion
   * onDelete callback - this function will be called when the delete action is 
   * triggered on a Gallery image
```

Reply:
Weird, I'm surprised this wasn't automatically fixed by the linter 🤔